docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column.scss

```diff
@@ -8,27 +8,29 @@
 
   &.hb-page-width {
     @include hb-page-width;
+    // scss-lint:disable ImportantRule
```

Comment:
I've added this in my PR, so I think you can remove all the new green stuff :) it will all be part of the hb-page-width mixin now